src/flag_gems/testing/__init__.py

```diff
@@ -9,6 +9,11 @@
     torch.bfloat16: 0.016,
 }
 
+RESOLUTION_DROPOUT = {
```

Comment:
is it used in the test?

Reply:
We will remove it along with "to_reference_gpu"